docs/index.rst

```diff
@@ -595,6 +678,50 @@ Users
        "last_name": "Watson"
      }
 
+.. http:post:: /password-resets/
```

Comment:
During our discussion I thought we'd create endpoints like:
```
.. http:create /password-resets/ (for creating a reset request)
.. http:post /password-resets/ (for updating the password and using up the reset)
```
It's a bit of a cludge, so I don't mind abandoning the RESTful structure and just having action endpoints like you've created, just double checking that you're intentionally going this route.